packages/dataviews/src/dataviews.tsx

```diff
@@ -51,22 +54,6 @@ const defaultGetItemId = ( item: ItemWithId ) => item.id;
 
 const defaultOnSelectionChange = () => {};
 
-function useSomeItemHasAPossibleBulkAction< Item >(
```

Comment:
The change does not seem related. Probably it was a rebase issue.

Reply:
It's unrelated but intentional. We had this duplicate function in the same package with no reason. I could remove from this PR, but I think it's harmless and super straightforward.

Reply:
Got it feel free to leave it.